### DISCUSSION TITLE
Add Right Click Menu Item to "Copy Stripped Link"

### DISCUSSION THREAD

**Discussion by ScottzCodez** [ClearURLs/Addon#331] [Ideas]
I would like to be able to right click a link and strip it before putting it on the clipboard.
I often copy links into my notes. but the Redirect tracking links in youtube video description section makes it very long.   
I have to click the link,, then copy the short url from the new page that loads
-- Another feature that would be nice is to Copy multiple text and links to the clipboard while stripping the tracking from the links.
Maybe it could be all in one menu item called "Copy text and strip links"

**Comment by KevinRoebert** [comment]
Hi. There is already a right click feature implemented.

**Comment by ScottzCodez** [comment]
It's not working on firefox v119. What is the Name of the menu item?

**Reply by KevinRoebert** [reply]
<img width="418" alt="grafik" src="https://github.com/ClearURLs/Addon/assets/28713602/247b52b1-3063-4fed-84c5-601c3f52a15a">

**Reply by ScottzCodez** [reply]
I don't have that on firefox on windows.  Yet the option is ON in extension settings "Display context menu entry"
